app.ts

```diff
@@ -255,6 +262,10 @@ function FormatEBike(eb: EBike): string {
                         const $span = document.createElement('span');
```

Comment:
Remove the `map` above and `FormatEBike()`, then:
```js
forEach(ebike => {
const $span = document.createElement('span');
const $name = document.createElement('span');
$span.appendChild($name);
$name.textContent = ebike.name;

if (SHINY_BIKES.includes(ebike.name)) { $name.classList.add("text-shiny"); }

if (ebike.battery != -1) {
  const $bat =  document.createElement('span');
  $bat.textContent = ` (${ebike.battery}%)`;
  $span.appendChild($bat);
}

$best_ebikes.appendChild($span);
})
```

Reply:
Done (small changes to make it actually work).